auslib/test/admin/views/test_rules.py

```diff
@@ -813,54 +1607,47 @@ def testDeleteWithoutPermission(self):
 
 
 class TestRuleHistoryView(ViewTest):
-    def testGetNoRevisions(self):
-        url = "/rules/1/revisions"
```

Comment:
Ah, I removed this because AFAIK there's no valid situation where a rule should have zero revisions. I can switch it to another rule if you think the test has value though.

Reply:
Ah, bit of a brain fade by me there. Never mind!